src/main/kotlin/calculator/Delimiter.kt

```diff
@@ -0,0 +1,20 @@
+package calculator
+
+class Delimiter {
+
+    fun getDelimitersOfText(text: String): List<String> {
+        val delimiters = mutableListOf<String>().apply{ addAll(DEFAULT_DELIMITERS) }
+        val customDelimiter = CUSTOM_DELIMITER_FLAG_REGEX.find(text)?.groupValues?.get(1)?.toList()
+        customDelimiter?.let {
+            require(it.all { char -> char.toString() != MINUS_OPERATOR })
+            delimiters.addAll(it.map { char -> char.toString() })
+        }
+        return delimiters
+    }
```

Comment:
코드에서 null-safe 연산을 많이 수행하고 계시네요!
이렇게 `?.` 가 계속해서 작성되는 코드는`elvis` 연산자만 잘 활용해주어도 가독성 있는 코드로 만들 수 있을 것 같아요🙂

```suggestion
fun getDelimitersOfText(text: String): List<String> {
        val customDelimiter = extractCustomDelimiter(text)
            ?: return DEFAULT_DELIMITERS

        return DEFAULT_DELIMITERS + listOf(customDelimiter)
    }

    private fun extractCustomDelimiter(text: String): String? {
        val matchResult = CUSTOM_DELIMITER_FLAG_REGEX.find(text) ?: return null
        val customDelimiter = matchResult.groupValues[1]
        require(customDelimiter != MINUS_OPERATOR)
        return customDelimiter
    }
```